blogs/ecs-ec2-crossvpc-with-tls/frontend-envoy.sh

```diff
@@ -0,0 +1,131 @@
+# Create YAML file #
+cat <<-"EOF" > /tmp/install_envoy.yml
+---
+schemaVersion: '2.2'
+description: Install Envoy Proxy
+parameters: 
+  region:
+    type: String
+  meshName:
+    type: String
+  vNodeName:
+    type: String 
+  ignoredUID:
+    type: String
+    default: '1337'
+  proxyIngressPort:
+    type: String
+    default: '15000'
+  proxyEgressPort:
+    type: String
+    default: '15001'
+  appPorts:
+    type: String
+  egressIgnoredIPs:
+    type: String
+    default: '169.254.169.254,169.254.170.2'
+  egressIgnoredPorts:
+    type: String 
+    default: '22'
+mainSteps:
+- action: aws:configureDocker
+  name: configureDocker
+  inputs:
+    action: Install
+- action: aws:runShellScript
+  name: installEnvoy
+  inputs:
+    runCommand: 
+      - |
+        #! /bin/bash -ex
+        sudo yum install -y jq
+        $(aws ecr get-login --no-include-email --region {{region}} --registry-ids 840364872350)
+        # Install and run envoy
+        sudo docker run --detach \
+          --env APPMESH_VIRTUAL_NODE_NAME=mesh/{{meshName}}/virtualNode/{{vNodeName}} \
+          --env ENABLE_ENVOY_XRAY_TRACING=1 \
+          --log-driver=awslogs \
+          --log-opt awslogs-region={{region}} \
+          --log-opt awslogs-create-group=true \
+          --log-opt awslogs-group=appmesh-workshop-frontend-envoy \
+          --log-opt tag=ec2/envoy/{{.FullID}} \
+          -u {{ignoredUID}} --network host \
+          840364872350.dkr.ecr.{{region}}.amazonaws.com/aws-appmesh-envoy:v1.15.1.0-prod
```

Comment:
Same as other comments, should be parameterized (environment variable is probably fine for this).

Reply:
added parameter